SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# RIOT Security Policy
+
+All security bugs reported will be silently fixed in `master` and backported to the
+previous release.
+
+## Reporting a Vulnerability
+
+If a security issue is discovered, please report it to security@riot-os.org.
+A response will be provided within one week.
+The issue will be tracked in the mailing list.
```

Comment:
be more specific. which mailing list?

Reply:
I thought security@riot-os.org if it is private.

Reply:
then: will be tracked via security@riot-os.org. ... or something like this.